schemas/gazette-notice-schema.json

```diff
@@ -0,0 +1,171 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Notice",
+  "description": "A gazette notice",
+  "type": "object",
+  "properties": {
+    "issue": {
+      "description": "The issue in which the notice is published",
+      "$ref": "#/definitions/issue"
+    },
+    "identifier": {
+      "description": "The identifier of the notice",
+      "type": "string"
+    },
+    "title": {
+      "description": "The title of the notice",
+      "type": "string"
+    },
+    "description": {
+      "description": "A description of the notice",
+      "type": "string"
+    },
+    "body": {
+      "description": "The content of the notice",
+      "type": "string"
+    },
+    "classification": {
+      "description": "The type of the notice",
```

Comment:
As noted above, unless otherwise noted, all properties are the gazette's. If will be very redundant to explain every time which properties are the gazette's vs calculated by OC.